### PR TITLE
Add SystemPulse with CPU, Memory and Disk usage stats

### PR DESCRIPTION
**Pull request by ajay-gov**
Have confirmed that this works, was able to pull system pulse from `swim:meta:mesh` and `swim:meta:part`

Code duplication in `MeshTable` and `PartTable` as well, open to suggestions on refactoring this piece.
Using 2 seconds as the poll interval in `MeshTable` and `PartTable` for getting the `SystemPulse`- Is that okay?

### CODE REVIEW

**Comment by c9r** [comment]
I would remove the timers from MeshTable and PartTable. The `collectProfile` methods are already throttled, so no need for the extra timers. I would also consolidate the two `updateSystemPulse` methods into a static `SystemPulse.latest()` method.

For extra efficiency, you could debounce the `SystemPulse` latest method to only collect new metrics when at least some interval has passed since the previous update, similar to how `collectProfile` uses `lastReportTime` to debounce updates. That way we're guaranteed to never gather system metrics more than some fixed rate, without introducing any new timers.

**Comment by ajay-gov** [comment]
Sounds good, will make those changes.

**Comment by ajay-gov** [comment]
@c9r Addressed these comments with the latest commit

File: swim-java/swim-runtime/swim-host/swim.system/src/main/java/swim/system/router/PartTable.java [hunk 1]
```diff
@@ -881,6 +882,7 @@ protected PartProfile collectProfile(long dt) {
   }
 
   static final Uri HOSTS_URI = Uri.parse("hosts");
+  static final long POLL_INTERVAL = 2000L;
```
Comment (c9r) [inline swim-java/swim-runtime/swim-host/swim.system/src/main/java/swim/system/router/PartTable.java:885]: This line can disappear now.

**Review by c9r — APPROVED** [review]
One lingering `POLL_INTERVAL` constant. Other than that, LGTM.

**Review by brohitbrose — APPROVED** [review]

**Pull request merged**